src/pybind/mgr/restful/module.py

```diff
@@ -227,12 +227,17 @@ def _serve(self):
             separators=(',', ': '),
         )
 
+        ssl_context = (
+            '/etc/ceph/ceph-mgr-restful.crt',
```

Comment:
This path should be something configurable so that we can do things like make vstart.sh work, and also perhaps so that people can run two mgr instances on the same node with different certs.

The convenient place from a runtime point of view would be in ``mgr_data``, but of course at package installation time that location is unknown.

What if the module looked for certs in both locations, so that anyone who wanted a per-daemon cert could put it in the /var/lib... location, and otherwise would get the global one generated at installation time?